test/UnitTests/MSTestAdapter.UnitTests/Discovery/TypeEnumeratorTests.cs

```diff
@@ -98,6 +98,23 @@ public void GetTestsShouldReturnBaseTestMethodsInSameAssembly()
 
     public void GetTestsShouldReturnBaseTestMethodsFromAnotherAssemblyByDefault()
     {
+        string runSettingsXml =
+            """
+            <RunSettings>
+              <MSTestV2>
+                <CaptureTraceOutput>true</CaptureTraceOutput>
```

Comment:
this test was failing when all tests run and passing when run by itself, it inherited the correct setup from somewhere previously. These are the exact settings it needs, copied from a test that is just below it.